sycl/test-e2e/AbiNeutral/catch-exception.cpp

```diff
@@ -0,0 +1,53 @@
+// RUN: %{build} -o %t.out
+// RUN: %{run} %t.out
+// RUN: %if preview-breaking-changes-supported %{ %{build} -fpreview-breaking-changes -D_GLIBCXX_USE_CXX11_ABI=0 -o %t2.out %}
+// RUN: %if preview-breaking-changes-supported %{ %{run} %t2.out %}
+// REQUIRES: level_zero && gpu
```

Comment:
Could you please add a comment at the beginning of each test describing what they intend to test? It is unclear to me how these relate to a "neutral" ABI.

Reply:
The test itself is nothing special other than the compiler flag -D_GLIBCXX_USE_CXX11ABI=0.
This did not work before since SYCL did not allow pre-C++11 ABI user programs.

Reply:
Done. Added comments.